src/server/services/afis/afis-facturen.ts

```diff
@@ -273,28 +274,24 @@ function transformFacturen(
   };
 }
 
-function shouldFilterOutFactuur(postingDate: string): boolean {
-  const postingDateAsDate = new Date(postingDate);
+function isDownloadAvailable(postingDate: string): boolean {
```

Comment:
Kun je ook nog even in een comment erbij zetten waarom we deze specifieke filtering toepassen?

Reply:
Is die ik er nu heb gezet goed? Ik twijfel over een specifieke tijd erin vermelden, maar het is misschien wel duidelijk zo